proposed/0008-Connection_management.md

```diff
@@ -0,0 +1,82 @@
+- Feature Name: Improved Connection Management
+- Type: Enhancement
+- Related Components: routing, crust
+- Start Date: 23-09-2015
+- RFC PR: (leave this empty)
+- Issue Number: (leave this empty)
+
+# Summary
+
+Introduce the notion of clearly defined connection phases and establish directed routing network connections in conjunction with crust.
+
+# Motivation
+
+Both outgoing and incoming connections are created to peers with unknown direction. Use the connect/accept connection code available from crust to store timed state in-line with crust connection handling changes for connections in either direction. This will allow us to determine unambiguously who the connection initiator is and act accordingly.
```

Comment:
I don't think this really explains why we need/want this functionality.  There's no reason given why keeping the state and knowing direction of connections is a good thing or solves an existing issue.